pkg/oc/admin/node/node_options.go

```diff
@@ -92,7 +92,6 @@ func (n *NodeOptions) Complete(f kcmdutil.Factory, c *cobra.Command, args []stri
 		return kcmdutil.PrinterForOptions(kcmdutil.ExtractCmdPrintOptions(c, withNamespace))
 	}
 
-	n.CmdPrinterOutput = true
```

Comment:
what on earth is this?

Reply:
it appears to only be used in listpods.go
  - https://github.com/openshift/origin/blob/master/pkg/oc/admin/node/listpods.go#L36
  - https://github.com/openshift/origin/blob/master/pkg/oc/admin/node/listpods.go#L47

It was being set to false a couple of lines above, and set to true here. Not sure why, but setting it to true was causing humanreadable output for pods to be printed without NAMESPACE (and other columns)